src/videojs-http-streaming.js

```diff
@@ -433,7 +434,13 @@ class HlsHandler extends Component {
       this.options_.bandwidth === Config.INITIAL_BANDWIDTH;
 
     // grab options passed to player.src
-    ['withCredentials', 'limitRenditionByPlayerDimensions', 'bandwidth', 'smoothQualityChange'].forEach((option) => {
+    [
+      'withCredentials',
+      'limitRenditionByPlayerDimensions',
+      'bandwidth',
+      'smoothQualityChange',
+      'customTagParsers'
```

Comment:
We should add some documentation on how to provide this option

Reply:
Also, don't forget that this change, and changes like, affect Video.js as a whole as well.